docs/articles/sde/introduction_to_sde.rst

```diff
@@ -16,6 +16,9 @@ Dependencies:
 
 * Vagrant
 * VirtualBox
+* Self-extracting SDK package to install into the SDE. `See, Building the SDK installer`_.
+
+.. _`See, Building the SDK installer`: http://pelux.io/software-factory/master/swf-blueprint/docs/articles/baseplatform/creating-sdk.html#building-the-sdk
```

Comment:
See

Reply:
Linking internal pages should use references. Instead of giving the link, you can just reference with :ref:`building-the-sdk` (the ` sign is converting the text to code)